pylint/checkers/base.py

```diff
@@ -1731,6 +1731,12 @@ class ComparisonChecker(_BasicChecker):
                       'comparison-with-itself',
                       'Used when something is compared against itself.',
                       ),
+            'W0143': ('Comparing bare callable, might have skipped parenthesis after callable',
```

Comment:
I'd just rephrase this as `Comparing against a callable, did you omit the parens?` I'm not entirely sold on the part after the comma, since we don't usually have messages that *talk* with the user like that, but can't think of something more suggestive.

Reply:
@PCManticore all points noted down. I will raise PR again in a day or two.
For msg, how about "Comparing against a callable, check parenthesis after callable"
Although did you omit... also seems correct to me as both messages points towards a probable root cause of the issue.